crawl/dependency.py

```diff
@@ -216,6 +216,10 @@ def bazel_label_name(self):
             name = "@%s//:%s" % (self._maven_install_name, name)
         return name
 
+    @property
+    def bazel_dep_name(self):
+        return self._bzl_artifact_name()
+
```

Comment:
Why is this needed?

Reply:
To search in override file. The `bazel_label_name` property adds `_maven_install_name` to the artifact name.
Example: Override file has mapping like -> `"javax.activation:activation": "@jakarta//:jakarta_activation_jakarta_activation_api"`

`bazel_label_name` property returns `@javax//:javax.activation:activation`
'bazel_dep_name` property returns `@javax//:javax.activation:activation`

Reply:
They don't both return the same value?

Reply:
No
![Uploading Screenshot 2023-10-13 at 10.51.59 AM.png…]()

Reply:
I mean, in the example above that you have they seem to both return the same value.